package.json

```diff
@@ -71,6 +71,8 @@
     "coffeescript": "^2.0.3",
     "cross-env": "^5.1.1",
     "eslint": "^4.13.0",
+    "glsl-token-assignments": "^2.0.2",
```

Comment:
Apparently this is even a bug in the current master branch if looking at the test results